src/manifolds/ProductManifold.jl

```diff
@@ -129,7 +129,7 @@ end
     check_manifold_point(M::ProductManifold, p; kwargs...)
 
 Check whether `p` is a valid point on the [`ProductManifold`](@ref) `M`.
-If `p` is not a point on `M` a [`CompositeManifoldError`](@ref) consisting of all error messages of the
```

Comment:
What's the problem with that `@ref`?

Reply:
Could work, I think since it moved to ManiofldsBase, for some time it might not have worked.